test/DisplayCervicalCancerMedicalHistory/cases/InsufficientResult.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Test code for Specimen unsatisfactory for diagnosis
+
+externalData:
+- resources
+
+data:
+-
+  resourceType: Patient
+  name: Joanne Smith
+  gender: female
+  birthDate: 1980-01-01
+  extension:
+  -
+    url: http://hl7.org/fhir/us/core/StructureDefinition/us-core-birthsex
+    valueCode: F
+- 
+  resourceType: DiagnosticReport
+  code: LOINC#65753-6 Cervix Pathology biopsy report
+  status: final
+  conclusionCode:
+  - SNOMEDCT#112631006 Specimen unsatisfactory for diagnosis (finding)
+  effectiveDateTime: 2021-05-01
+
+results:
+  HistologyDiagnosticReportsSummary: [{value: 'Insufficient/Unsatisfactory'}]
```

Comment:
Okay, since your suggestion to use `label` from Mike's PR merged in, I will have to change the test script anyway. I will change and try again.

Reply:
This thread is no longer relevant, since you added a new test in https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/commit/eb470ced8ff011e2304032de07459e2e67f778ed where you are testing against the `MostRecentBiopsyResult` CQL expression, which returns just a string.